### PR TITLE
Feature/mental health views

### PR DESCRIPTION
**Pull request by petermason**
# Description

* Mental health study views
* All state is managed via redux and redux persist

## On what platform have you tested the change?

- [x] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

image

image

image

image

image

image

image

image

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/features/dashboard/DashboardScreen.tsx [hunk 1]
```diff
@@ -128,6 +128,7 @@ export const DashboardScreen: React.FC<Props> = ({ navigation, route }) => {
       compactHeader={<CompactHeader reportOnPress={onReport} />}
       expandedHeader={<Header reportOnPress={onReport} />}>
       <View style={styles.calloutContainer}>
+        <Button title="Mental Health Study" onPress={() => appCoordinator.goToMentalHealthStudy()} />
```
Comment (bionara) [inline src/features/dashboard/DashboardScreen.tsx:131]: worth having a translation key to avoid pain down the line?

Comment (petermason) [reply src/features/dashboard/DashboardScreen.tsx:131]: This is PR is a wip :)

Comment (bionara) [reply src/features/dashboard/DashboardScreen.tsx:131]: jut proving i read it lol. i know this is for UK only too btw!

**Review by bionara — APPROVED** [review]
awesome work!

**Pull request merged**